.github/workflows/pdm-lock-automation.yaml

```diff
@@ -66,30 +71,15 @@ jobs:
           fi
           source .venv/bin/activate
           pdm lock -G :all
```

Comment:
```suggestion
          pdm lock -G :all -v
```
Making it verbose to quickly analyze any issues

Reply:
done @chandrasekharan-zipstack